app/styles/app.scss

```diff
@@ -60,14 +70,28 @@ $purple: rgb(167, 41, 235);
 // Medium Devices, Desktops
 @media only screen and (min-width : 992px) {
   .btn-danger {
-    width:180px;
-    height: 100px;
+    width:130px;
+    height: 55px;
+    margin-left: 5px;
+    margin-right: 5px;
+    margin-top: 18px;
+    margin-bottom: 18px;
   }
 
   #logo{
-    width: 220px;
-    height: 100px;
+    width: 200px;
+    height: 95px;
+  }
+  #footer {
+  padding-top: 50px;
+  text-align: left;
+  a{
+    color:black;
   }
+  center{
+    color:black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space